packages/client/tests/e2e/custom-output-yarn/readme.md

```diff
@@ -0,0 +1,11 @@
+# Readme
+
+This tests ensures that custom output clients are working as expected with Yarn.
```

Comment:
```suggestion
This tests ensures that custom output clients are working as expected with Yarn 1.
```
I'm not sure how this goes with other versions

Reply:
Same with Berry